src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Microsoft.VisualStudio.LanguageServerClient.Razor.ruleset

```diff
@@ -70,10 +70,16 @@
   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
     <Rule Id="IDE0060" Action="None" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.NetAnalyzers">
```

Comment:
Changes to this file enable [this](https://github.com/dotnet/aspnetcore-tooling/pull/2665/files#diff-39d22959babee2b6a609d1afef8dc6a4dd7f1af00158abf6c6c4145291be4ae9R27-R31) without warnings.